### ISSUE TITLE
New prebid version doesn't show all bids in the console

### ISSUE THREAD

**Issue by GiovanniPascoli** [prebid/Prebid.js#2640]
## Type of issue
Request

## Description
Upgraded to Prebid v1.12.0 the snippet code to see all bids in the console, now show only winning or looser bids and not all like preceded version (which was also showing the no bid available responses).
Is there a way to restore the previous behavior, even by using undocumented APIs?

## Test page
None
## Platform details
Prebid v1.12.0 Chrome, ecc.

## Other information
I follow steps on the "See all bids in the console" section on 
http://prebid.org/dev-docs/toubleshooting-tips.html

**Comment by jnsprngs** [comment]
Seconding this issue.

I am testing `v1.12.0-pre` and not seeing any bids with a `statusMessage : 'Bid returned empty or error response'` instances. Instead, it looks like null bid responses are excluded from `pbjs.getBidResponses()`.

**Comment by mercuryyy** [comment]
We all second this issue, but this is within prebid new core. @prebid Team mentioned in another thread they do not have time to address this and its open if anybody want to submit a pull.

I think getBidResponses() should return all bids, so we can see who timed out and who was within the timeout but returned no bid.

**Comment by mkendall07** [comment]
yes open for anyone to submit a pull request but I don't think the work is trivial.

**Comment by YOzaz** [comment]
This is required imho.
Google Analytics module now fails to identify which response has failed or timed out.

**Comment by dejanstrbac** [comment]
It is not really prebid that filters the zero cpm bids that previously used to come through as "empty or error response".

Prebid now tags all bids coming from the adapters as "available", even if cpm=0. It is adapters that are also now ignoring own 0 cpms.

One way to get similar data now could be by listening to bidderDone event. You can then deduct a zero cpm bid. For timeouts, you could use the same event (well, the lack of it) or the bidderTimeout event which is not reliable, but it will be impossible to get the information of the final TTR as in Prebid < 1.0.

**Comment by Tigerroad** [comment]
Is this also related to Google Analytics labels not being set for EventAction: Timeouts. I was just about to open a new Issue but it seems related?

**Comment by andyblackwell** [comment]
Here's a start at it. It's working for my needs, but I haven't fully tested. Maybe it could point someone with enough time to do a proper PR in the right direction?
https://gist.github.com/andyblackwell/8815a2b721ad82fbe450347fa221471e
diff with v1.14.0: https://www.diffchecker.com/FtVTQlet

**Comment by andyblackwell** [comment]
bumping out of stale. Can this be labeled somehow so it doesn't get auto-closed?

**Comment by andyblackwell** [comment]
Can we get this labeled as a bug or feature, so it stops getting set to stale? @mkendall07

**Comment by mike-chowla** [comment]
I'm reviewing #3022 but wondering if makes sense to change the core to solve the problem of debugging output.  Shouldn't the debug snippet just be changed to handled this case?
